docs/getting-started.md

```diff
@@ -1,27 +1,17 @@
-# Tekton Pipelines Tutorial
+# Getting Started with Tekton Pipelines
 
-This tutorial uses a simple `Hello World` example to show you how to:
-- Create a `Task`
-- Create a `Pipeline` containing your `Tasks`
-- Use a `TaskRun` to instantiate and execute a `Task` outside of a `Pipeline`
-- Use a `PipelineRun` to instantiate and run a `Pipeline` containing your `Tasks`
+This guide will get you started with tekton pipelines with a simple `Hello World` tutorial.
 
-This tutorial consists of the following sections:
+## Before we begin
 
-- [Creating and running a `Task`](#creating-and-running-a-task)
-- [Creating and running a `Pipeline`](#creating-and-running-a-pipeline)
+Before we begin this tutorial, make sure you have [installed and configured](https://github.com/tektoncd/pipeline/blob/master/docs/install.md)
```

Comment:
"You" instead of "we". It's the reader completing the steps, not someone else together with them.